cli/kubectl_kadalu/install.py

```diff
@@ -39,6 +40,21 @@ def validate(_args):
 
 def run(args):
     """ perform install subcommand """
+
+    # Check if kadalu operator is present already.
+    cmd = utils.kubectl_cmd(args) + ["get", "deployments", "-nkadalu", "-ojson"]
+    resp = utils.execute(cmd)
```

Comment:
This can raise two exceptions. 1) `utils.CommandError` when command fails, 2) `FileNotFoundError` when kubectl binary is not found. Ref: https://github.com/kadalu/kadalu/blob/devel/cli/kubectl_kadalu/storage_add.py#L151-L154